src/main/kotlin/io/sc3/goodies/misc/StairWrenchItem.kt

```diff
@@ -0,0 +1,88 @@
+package io.sc3.goodies.misc
+
+import io.sc3.goodies.util.BaseItem
+import io.sc3.text.color
+import net.minecraft.block.Block
+import net.minecraft.block.BlockState
+import net.minecraft.block.StairsBlock
+import net.minecraft.client.item.TooltipContext
+import net.minecraft.entity.player.PlayerEntity
+import net.minecraft.item.ItemStack
+import net.minecraft.item.ItemUsageContext
+import net.minecraft.server.network.ServerPlayerEntity
+import net.minecraft.state.property.Property
+import net.minecraft.text.Text
+import net.minecraft.text.Text.translatable
+import net.minecraft.util.ActionResult
+import net.minecraft.util.Formatting
+import net.minecraft.util.Hand
+import net.minecraft.util.Util
+import net.minecraft.util.math.BlockPos
+import net.minecraft.world.World
+import net.minecraft.world.WorldAccess
+
+class StairWrenchItem(settings: Settings) : BaseItem(settings) {
+  private var properties: List<Property<*>> = listOf(StairsBlock.FACING, StairsBlock.HALF, StairsBlock.SHAPE)
```

Comment:
back to this one, is it possible to remove the type entirely? either:
```suggestion
  private var properties = listOf(StairsBlock.FACING, StairsBlock.HALF, StairsBlock.SHAPE)
```

or if not:

```suggestion
  private var properties = listOf<Property<*>>(StairsBlock.FACING, StairsBlock.HALF, StairsBlock.SHAPE)
```

just trying to think of making this line less verbose

Reply:
the linter doesn't yell at me so presumably not, will test.